lib/miq_automation_engine/service_models/miq_ae_service_tenant.rb

```diff
@@ -0,0 +1,17 @@
+module MiqAeMethodService
+  class MiqAeServiceTenant < MiqAeServiceModelBase
+    expose :id
+    expose :domain
+    expose :subdomain
+    expose :name
+    expose :login_text
+    expose :logo_file_name
+    expose :logo_content_type
```

Comment:
don't expose logo_content_type or logo_file_size

I wonder if we want to just assign a photo to this model and handle these fields internally.

@Fryguy thoughts?